src/lib.rs

```diff
@@ -108,6 +111,14 @@ impl Config {
         self.max_num_streams = n
     }
 
+    /// Set the max. number of pending frames, i.e. frames
+    /// which are to be sent to the remote, but can not right
+    /// away and need to be buffered.
```

Comment:
Is it not the case that all frames to be sent go through the buffer, i.e. the `pending` queue? The comment may then be slightly misleading as one may think that even having a 0-sized buffer would be a valid configuration, only causing errors when a frame cannot be sent "right away" and needs to be buffered. It may then be better to just say something like "[..], i.e. outgoing frames which have not yet been sent."? If a `0` value for this option is not sensible, then using `NonZeroUsize` at least for the parameter may also be worth considering.